### DISCUSSION TITLE
Generation of indented mixed lists from markdown to docx

### DISCUSSION THREAD

**Discussion by zero0n3** [jgm/pandoc#9990] [General]
Greetings
I would like to generate mixed lists of numbered lists and bulleted, indented lists in markdown to be converted into docx.
However, both with the markdown extension and with the commonmark I cannot get the numbering to resume from the second point, that is.
This is the markdown i write:

```
1. Dog
    * German Shepherd
    * Belgian Shepherd
        * Malinois
        * Groenendael
        * Tervuren
2. Cat
    * Siberian
    * Siamese
 ```
The first numered list work and also the bullet nested lists work but when it reaches the point numbered "2.", where it should resume the numbering with "2.", a bullet appears instead of the number.

![2024-07-16 09_12_58-RCC003788_bozza docx  Modalità di compatibilità  - Word](https://github.com/user-attachments/assets/9ae9a4b7-041b-4927-a46b-ed15cf6d727a)

Where ma I wrong?
thanks

**Comment by jgm** [comment]
This looks like a bug, and indeed, a regression.  (I just confirmed that I get correct output with pandoc 3.1.1.)

**Reply by jgm** [reply]
3.2 also gives correct output.  **This is a regression in 3.2.1.**

**Reply by jgm** [reply]
I would suspect the fix for #5198 perhaps...
https://github.com/jgm/pandoc/commit/f5531f18a54ca008eaa7caadb216657d28a7ffa5

**Reply by jgm** [reply]
Minimal repro:
```
1. one
   - bullet
2. two
```

**Reply by jgm** [reply]
I have confirmed that the bug was introduced by commit
https://github.com/jgm/pandoc/commit/f5531f18a54ca008eaa7caadb216657d28a7ffa5

**Reply by positivepauly** [reply]
We are having the same issue. Tried a bunch of different MD coding. Works fine converting to PDF and HTML, but DOCX seems to continue the format of the previous item's sub-item in a numbered/lettered list.

**Reply by jgm** [reply]
I pushed a fix last night.  You can try compiling from source or trying tomorrow's nightly.

**Comment by jgm** [comment]
see #9994

**Comment by zero0n3** [comment]
solved in 3.3. thanks
